content/applications/finance/fiscal_localizations/ecuador.rst

```diff
@@ -650,6 +650,97 @@ electronic invoice after the checkout is completed.
 After finishing the checkout process, a confirmed invoice is generated, ready to be sent manually or
 asynchronously to the SRI.
 
+Point of Sale electronic invoicing
+----------------------------------
+
+:ref:`Install <general/install>` the *Ecuadorian Module for Point of Sale* (`l10n_cl_edi_pos`)
+module to enable the following features and configurations:
+
+- Choose the SRI payment method in each payment method configuration.
+- Customers can manually input their identification type and identification number when creating a
+  new contact on *POS*.
+- Automatically generate a valid electronic invoice for Ecuador at the end of the checkout process.
+
+Configuration
+~~~~~~~~~~~~~
+
+Payment method
+**************
+
+To set a payment method for a point of sale, go to :menuselection:`POS --> Configuration --> Payment
+Methods --> Set SRI Payment Method`. Set the :guilabel:`SRI Payment Method` in the payment method
+for each point of sale.
+
+QR code POS receipt
+*******************
+
+To activate the functionality of a QR code on the POS receipt, navigate to :menuselection:`POS -->
+Configuration --> Settings` to find the :guilabel:`Activate QR Code` option.
+
+Enabling this allows the client to scan the code and access the invoice from the POS order from the
+Odoo portal.
+
+.. note::
+   If the client requests a credit note due to a return of this type of purchase, the credit note
+   and return process can be managed directly from the POS session.
+
+Invoicing flows
+~~~~~~~~~~~~~~~
+
+Type and identification number
+******************************
+
+The POS cashier will be able to create a new contact for a customer who requests an invoice from the
+sale. For this process, two new fields have been added to the contact creation form. Type of
+identification and identification number.
+
+.. note::
+   A verification exists to verify that the identification number filled has the correct length. For
+   RUC identification, 13 digits are required. For Cédula, 9 digits are required. When the
+   configurations are all set, this feature will cover the next invoicing flows.
```

Comment:
A verification exists to verify sounds weird to me; you also mention a feature: do you mean a feature checks the identification number length? 
Also, I get that Cédula and RUC are mentioned as examples? It looks like a very short list. Or maybe the verification feature only covers these two?

Reply:
```suggestion
As the identification number length differs depending on the identification type, Odoo automatically checks the :guilabel:`Tax ID` field upon saving the customer form. To manually ensure the length is correct, know that the :guilabel:`RUC` and :guilabel:`Citizenship` types require 13 and 10 digits respectively.

```